web/turkey/survey/static/survey/js/auditors/clicks_specific.js

```diff
@@ -0,0 +1,29 @@
+var AuditorClicksSpecifics = {
+    clicks_specifics: [],
+    log_click_content: function (e) {
+        e.stopPropagation();
+        this.clicks_specifics.push(e.target);
```

Comment:
e.target needs to be processed further either in log_click_content or submit_callable  (via mapping array). Not sure if you can even post a part of the DOM, but if you can, it will vary by browser. Basically, submit_callable should only be returning primitives: plain old objects (no prototypes and dom objects!), arrays, ints, strings, etc.